gdcdictionary/schemas/time_series.yaml

```
@@ -44,17 +44,22 @@ properties:
   confirmed:
     description: >
       Confirmed cases
-    type: string
+    type: integer
 
   deaths:
     description: >
       Total number of deaths
-    type: string
+    type: integer
     
   recovered:
     description: >
       Total number of recovered cases
-    type: string
+    type: integer
+  
+  testing:
+    description: >
+      Testing
```

According to the data source:
```suggestion
      Number of tests conducted
```